views/apartments/edit.erb

```diff
@@ -0,0 +1,10 @@
+<h2>Edit Apartment</h2>
+
+<form method="put" action="/apartments/<%= @apartment.id %>/update">
+  <input type="text" name="address" placeholder="Address" value="<%= @apartment.address %>">
```

Comment:
Here is where we'd change the form to 'namespace' the params, by changing the `name` attribute like so:
`<input type="text" name="apartment[address]" ....>`